doc/manual/package.nix

```diff
@@ -34,6 +34,7 @@ mkMesonDerivation (finalAttrs: {
       (fileset.unions [
         ../../.version
         # For example JSON
+        ../../src/libutil-tests/data/memory-source-accessor
         ../../src/libutil-tests/data/hash
```

Comment:
Maybe these should point the other way around?
Being a documentation example is more significant than being tested as part of some test suite.
Also it's helpful for the test suite to have an explicit reference to path that's in the manual, to distinguish it from test cases that only exist for the purpose of testing, i.e. corner cases that aren't relevant enough for the manual.

Reply:
It will write through the symlink *if* the symlink already exists, but I like to use `_NIX_TEST_ACCEPT=1` for making the initial versions of those files too.